docs/application/workflow/application-workflowConfig.json

```diff
@@ -37,21 +37,22 @@
               "action": "REQUEST_RESPONSE",
               "nextState": "PENDING_RESPONSE_APPROVAL",
               "roles": [
-                "APPLICATION_RESPONDER"
+                "SUBMISSION_RESPONDER"
               ]
             },
             {
               "action": "DELETE",
               "nextState": "DELETED",
               "roles": [
-                "APPLICATION_DELETE"
+                "SUBMISSION_DELETE"
```

Comment:
This is again an UPDATE to set the isActive flag to true in the DB. Why is this a separate role? This will be a SUBMISSION_EDITOR.

Reply:
this file will be removed